tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraLongLivedTokenSetMethodContract.scala

```diff
@@ -633,4 +633,60 @@ trait LinagoraLongLivedTokenSetMethodContract {
       .isTrue
   }
 
+  @Test
+  def methodShouldRevokeToken(): Unit = {
+    val createRequest: String =
+      s"""{
+         |  "using": ["urn:ietf:params:jmap:core", "com:linagora:params:long:lived:token"],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "create": {
+         |        "K38": {
+         |          "deviceId": "My android device"
+         |        }
+         |      }
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin
+
+    val createResponse: String = `given`
+      .body(createRequest)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    val longLivedTokenResponse: String = (((Json.parse(createResponse) \\ "methodResponses" )
+      .head \\ "created")
+      .head \\ "token")
+      .head.asInstanceOf[JsString].value
+
+    val destroyRequest: String =
+      s"""{
+         |  "using": ["urn:ietf:params:jmap:core", "com:linagora:params:long:lived:token"],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "destroy": ["$longLivedTokenResponse"]
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin
+
+    val destroyResponse: String = `given`
+      .body(destroyRequest)
+    .when()
+      .post().prettyPeek()
```

Comment:
I know debug... will remove it when I fix the issue here:
```
"error",
            {
                "type": "invalidArguments",
                "description": "{\"errors\":[{\"path\":\"obj.create\",\"messages\":[\"error.path.missing\"]},{\"path\":\"obj.destroy[0]\",\"messages\":[\"LongLivedTokenId does not match Id constraints: Predicate failed: 'bob@domain.tld_9d36a5b3-e8e0-4f3e-a052-f538a270b241' contains some invalid characters. Should be [#a-zA-Z0-9-_] and no longer than 255 chars.\"]}]}"
            },
```
bad copy/paste logic, a long lived id token is not a UUID

Reply:
Need more tests too